packages/fuel-gauge/src/experimental-contract.test.ts

```diff
@@ -108,4 +108,21 @@ describe('Experimental Contract', () => {
 
     expect(configurableValue).toStrictEqual(configurableParam);
   });
+
+  it('multicalls', async () => {
+    const { value: firstResults } = await contractInstance
+      .multiCall([contractInstance.functions.first_call(255)])
```

Comment:
what's the significance of these values? i.e. `255` and `555` ?

Reply:
Mostly arbitrary values. `255` as `first_call` takes a `u8` and `255` being max `u8`, and `555` as `second_call` takes `u16` so just picked a figure bigger than `255`. Used this set up as I just wanted arguments that used a different amount of bytes so it would be obvious on the byte-code which was which.

I used the `experimental-*` test suite as a proving ground for the new encoding. I had an issue with multi-call and this was a simple setup to fix it. It probably doesn't add any additional value to our existing multi-call test suite. But I'm going to either move or remove these tests in #2109.